server/graphql/v2/query/collection/ActivitiesCollectionQuery.ts

```diff
@@ -18,12 +18,18 @@ const ActivitiesCollectionArgs = {
   limit: { ...CollectionArgs.limit, defaultValue: 100 },
   offset: CollectionArgs.offset,
   account: {
-    type: new GraphQLNonNull(AccountReferenceInput),
-    description: 'The account associated with the Activity',
+    type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
```

Comment:
We still want people to pass at least one account to filter by

```suggestion
    type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(AccountReferenceInput))),
```